From 6dea25b08f8eda058dd69afb52ea41e061676ddd Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Wed, 21 Sep 2022 08:43:35 +0300
Subject: [PATCH] [SW-101000] greco: update comment message

This patch update comment in preboot, when we are masking
pcie addr dec interrupt, that it will unmasked in linux.

Change-Id: I67aef3192a0f8b009f2002eb14168801f197a06f
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/arch/greco/common/common.c b/arch/greco/common/common.c
index ca7a241..604df9f 100644
--- a/arch/greco/common/common.c
+++ b/arch/greco/common/common.c
@@ -878,6 +878,7 @@
 	regmap_writel(BIT(0), mmPCIE_WRAP_AXI_INTR);
 	/*
 	 * Mask PCIE ADDR DEC interrupt arising due to LBW region access.
+	 * Unmask it in linux after interrupt registration.
 	 * Currently in greco, LBW region interrupts don't show in cause register.
 	 * TODO: Checking with HW team regarding this, until then mask it.
 	 */
